spec/services/checksum_validator_spec.rb

```diff
@@ -317,42 +317,41 @@
 
       before do
         # would result in a status update if the save succeeded
-        comp_moab.online_moab_not_found!
+        complete_moab.online_moab_not_found!
 
         # do this second since we save! as part of setup
-        allow(comp_moab).to receive(:save!).and_raise(ActiveRecord::ConnectionTimeoutError)
+        allow(complete_moab).to receive(:save!).and_raise(ActiveRecord::ConnectionTimeoutError)
       end
 
       it 'does not re-throw an ActiveRecord error we know how to deal with' do
-        expect { cv.validate_checksums }.not_to raise_error
+        expect { checksum_validator.validate_checksums }.not_to raise_error
       end
 
       it 'has a result code indicating the update failed' do
-        cv.validate_checksums
-        expect(cv.results.contains_result_code?(AuditResults::DB_UPDATE_FAILED)).to be true
+        checksum_validator.validate_checksums
+        expect(checksum_validator.results.contains_result_code?(AuditResults::DB_UPDATE_FAILED)).to be true
       end
 
       it 'does not have a result code indicating the update happened' do
-        cv.validate_checksums
-        expect(cv.results.contains_result_code?(AuditResults::CM_STATUS_CHANGED)).to be false
+        checksum_validator.validate_checksums
+        expect(checksum_validator.results.contains_result_code?(AuditResults::CM_STATUS_CHANGED)).to be false
       end
     end
   end
 
   describe '#validate_manifest_inventories' do
     it 'calls validate_manifest_inventory for each moab_version' do
-      sov1 = instance_double(Moab::StorageObjectVersion)
-      sov2 = instance_double(Moab::StorageObjectVersion)
-      sov3 = instance_double(Moab::StorageObjectVersion)
-      version_list = [sov1, sov2, sov3]
-      moab = instance_double(Moab::StorageObject, version_list: [sov1, sov2, sov3])
-      allow(cv).to receive(:moab).and_return(moab)
+      storage_object_version1 = instance_double(Moab::StorageObjectVersion)
```

Comment:
I think `_v1` is sufficiently clear, myself.  But don't care much either way.

Reply:
In most contexts I would agree. However, this is a `storage_object_version` so using the full name.